src/pages/html5-video-support.jsx

```diff
@@ -82,7 +82,7 @@ const SecondPage = () => (
           <TableRow>
             <TableCell header>HEVC (H.265)</TableCell>
             <SupportCell>No</SupportCell>
-            <SupportCell supported>18</SupportCell>
+            <SupportCell>No *</SupportCell>
```

Comment:
Might be worth adding a non-breaking space, otherwise, the `*` wraps and doesn't look as good.
```suggestion
            <SupportCell>No&nbsp;*</SupportCell>
```